synapse/handlers/_base.py

```diff
@@ -15,8 +15,6 @@
 import logging
 from typing import TYPE_CHECKING, Optional
 
-import synapse.state
-import synapse.storage
```

Comment:
if these were really unused... why didn't the linter pick them up? :thinking:

Reply:
I *think* because they're just importing the module (as opposed to `from foo import bar`, but I had the same thought...